src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+  moviesList: Movie[]
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie, moviesList }) => {
+  const [titleSearchQuery, setSearchField] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [errorMessage, setErrorMessage] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
follow convention for the boolean variables names

Reply:
Done